spec_mode/mirimrs/notebook1/MRS_Notebook1.ipynb

```diff
@@ -140,14 +140,6 @@
     "- mirisim does not add all of the necessary header keywords for the pipeline to know how to do background subtraction, identify source type, etc.  In order to get these APT-derived keywords correct they will need to be set manually."
```

Comment:
<p>Since this notebook is written to work with version 1.1.0 of <code>jwst</code>, it could be worthwhile to raise a warning if a user is running a different version. To do so, you could add <code>import warnings</code> to your first import cell and then create a new cell with something like this:</p><pre class="ql-syntax" spellcheck="false">if jwst.__version__ != '1.1.0':
&nbsp;&nbsp;&nbsp; warnings.warn(f"You are running version {jwst.__version__} of the jwst "
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; "module instead of the intended 1.1.0.")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
This required a little reorganizing (since it is the import steps that will fail first if the version is outdated), but has been added.